src/app.js

```diff
@@ -218,11 +218,13 @@ function getToolUrl() {
     const localNavElement = $(`#section-tabs ${baseSelector}`);
     const globalNavElement = $(`#menu ${baseSelector}`);
 
-    if (localNavElement.attr('href') && localNavElement.text().trim() === word) {
+    if (localNavElement.attr('href') && localNavElement[0].text.trim() === word) {
       url = localNavElement.attr('href');
       return true;
     } else if (globalNavElement.attr('href') && globalNavElement.find('.menu-item__text').text().trim() === word) {
-      url = globalNavElement.attr('href');
+      const toolPath = globalNavElement.attr('href').match(/\/(external_tools)\/[0-9]+/i)
+      const contextMatch = window.location.pathname.match(/^\/(courses|accounts)\/[0-9]+/i)
```

Comment:
this will be null if you're at the root url (the dashboard) which will cause the next line to fail I think? This also shouldn't be necessary in an account context, so you can check if they're in a course, and if not, fall back to using the global nav href.